specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json

```diff
@@ -5904,7 +5904,7 @@
           "ServerFarms"
         ],
         "summary": "List of metrics that can be queried for an App Service Plan",
-        "operationId": "ServerFarms_GetServerFarmMetricDefintions",
+        "operationId": "ServerFarms_GetServerFarmMetricDefinitions",
```

Comment:
I understand this is to fix a typo - but existing SDKs will be broken - you might have to wait to fix this in the next API version.

Reply:
Didn't notice the breaking change. I'll remove it from this PR